templates/suspenders_layout.html.erb.erb

```diff
@@ -4,6 +4,7 @@
   <meta charset="utf-8" />
   <meta name="ROBOTS" content="NOODP" />
   <meta name="viewport" content="initial-scale=1" />
+  <%%# change the title in config/locales/en.yml in the en.titles key %>
```

Comment:
Better yet,

`<%%# Configure default and controller/view specific titles in config/locales/en.yml. https://github.com/calebthompson/title#usage $>`